### PR TITLE
style: update logo

### PR DESCRIPTION
**Pull request by asadali145**
#### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/3339

#### What's this PR do?
Updates the bootcamps and open learning logos

#### How should this be manually tested?

- Verify the Header and Footer logo on the Homepage.
- Smoke test all pages.
- Verify the letter logo at `/letters/<uuid>/`
- Verify the payment history logo at `/applications/<application-id>/payment-history/`
- Verify the payment statement logo at `/statement/<statement_id>`
- Verify the certificate logos.
- Verify the email logos. Email logos are not loading locally. I have verified the source path.

#### Screenshots (if appropriate)
Homepage (Desktop -- New)
Opera Snapshot_2024-02-13_174509_bc odl local

Homepage (Mobile -- New)
Opera Snapshot_2024-02-13_174527_bc odl local

Payment Statement (Desktop -- New)
Opera Snapshot_2024-02-13_174646_bc odl local

Payment Statement (Mobile -- New)
Opera Snapshot_2024-02-13_174703_bc odl local

Application Payment History (Desktop -- New)
Opera Snapshot_2024-02-13_175607_bc odl local

Application Payment History (<Mobile -- New)
Opera Snapshot_2024-02-13_175619_bc odl local

Letter (Desktop -- New)
Opera Snapshot_2024-02-13_175743_bc odl local

Letter (Mobile -- New)
Opera Snapshot_2024-02-13_175754_bc odl local

Certificate Dual Logo (Desktop -- New)
Opera Snapshot_2024-02-16_191114_bc odl local

Certificate Dual Logo (Mobile -- New)
Opera Snapshot_2024-02-16_191141_bc odl local

Certificate Single Logo (Desktop -- New)
Opera Snapshot_2024-02-16_191335_bc odl local

Certificate Single Logo (Mobile -- New)
Opera Snapshot_2024-02-16_191404_bc odl local

Email logo
Screenshot 2024-02-13 at 6 12 09 PM

Favicon
Screenshot 2024-02-16 at 7 12 03 PM

### CODE REVIEW

**Comment by asadali145** [comment]
I could not test the certificates locally. I am debugging the issue. PR is ready for review.

**Comment by asadali145** [comment]
The seed_data command does not create a certificate index page; when there is no index page, the Certificate pages end up with 404.

**Review by pdpinch — CHANGES_REQUESTED** [review]
Thanks @asadali145 -- I have a couple of changes:

- The address in the footer should be 600 Technology Square instead of 77 Massachusetts Avenue
- Remove the "Massachusetts Institute of Technology" between the MIT Open Learning Logo and the Street Address
- The logo in the footer of the certificate needs to be updated. I think it should be the the "MIT Bootcamps Open Learning" logo
- On the certificate, I think the MIT logo should be red. I think you can download the red logo from https://brand.mit.edu/system/files/downloads/MIT-logo.zip

**Comment by asadali145** [comment]
> The address in the footer should be 600 Technology Square instead of 77 Massachusetts Avenue

I have updated it for the Site Footer, Email Footer, and Statement. Here is how it looks now.

Site Footer:
Screenshot 2024-02-07 at 1 06 01 PM

Statement Footer:
Screenshot 2024-02-07 at 1 06 21 PM

Email Footer:
Screenshot 2024-02-07 at 1 07 07 PM

File: cms/templates/certificate_page.html [hunk 1]
```diff
@@ -96,7 +96,7 @@ <h2>Congratulations, {{ learner_name }}!</h2>
             <div class="certificate-logo">
                 <a href="https://web.mit.edu/" class="mit"></a>
                 <a href="/" class="bc"></a>
-                <img src="{% static 'images/certificates/certificate-bootcamps-logo.png' %}" alt="Bootcamps">
+                <img src="{% static 'images/mit-ol-logo.svg' %}" alt="Bootcamps">
```
Comment (arslanashraf7) [inline cms/templates/certificate_page.html:99]: The MIT Open Learning logo in the [designs](https://www.figma.com/file/GFl0GZREDaWmdRytsNiET2/MIT-Logo-Header-%2F-Footer?node-id=7%3A2675&mode=dev) is bigger than how it appears in the certificate.
image

File: cms/templates/certificate_page.html [hunk 2]
```diff
@@ -44,7 +44,7 @@ <h2>Congratulations, {{ learner_name }}!</h2>
           <div class="certificate">
             <div class="certificate-head">
               <div class="certificate-b-logo">
-                <img src="{% static 'images/certificates/certificate-bootcamps-logo.png' %}" alt="Certificate Bootcamp Logo">
```
Comment (arslanashraf7) [inline cms/templates/certificate_page.html:47]: When we add the secondary or partner logo in the certificate, the design becomes a little out. e.g. Check https://bootcamp-rc.odl.mit.edu/certificate/ff9a1737-c9c1-4200-b684-55382a586ac7/

And how it looks with this PR:

image


Notice the spacing below the bootcamps logo

File: cms/templates/certificate_page.html
Comment (arslanashraf7) [inline cms/templates/certificate_page.html:1]: Shouldn't we use the black MIT Logo from https://brand.mit.edu/logos-marks/mit-logo at the bottom?

File: main/templates/footer.html [hunk 3]
```diff
@@ -51,11 +51,10 @@ <h2><a href="https://mitbootcamps.zendesk.com/hc/en-us/requests/new">Contact Us<
         </div>
         <div class="col-md-4">
           <div class="address-area">
-            <h3>MIT Bootcamps</h3>
+            <img class="open-learning-logo-img" src="/static/images/mit-ol-logo.svg" alt="MIT Open Learning Logo" />
```
Comment (arslanashraf7) [inline main/templates/footer.html:54]: The spacing between the logo and address should be increased:
image

**Review by arslanashraf7 — CHANGES_REQUESTED** [review]
Overall it looks good. Just left a couple of small design comments. I'd also recommend getting an eye on this PR from @mbilalmughal .

File: cms/templates/certificate_page.html (hunk 1 above)
Comment (asadali145) [reply cms/templates/certificate_page.html:99]: Good catch. Thanks!

File: cms/templates/certificate_page.html
Comment (asadali145) [reply cms/templates/certificate_page.html:1]: Are you talking about the Red logo? It was suggested by Peter https://github.com/mitodl/bootcamp-ecommerce/pull/1484#pullrequestreview-1866777694 and I discussed with Bilal.

File: main/templates/footer.html (hunk 3 above)
Comment (asadali145) [reply main/templates/footer.html:54]: Updated in https://github.com/mitodl/bootcamp-ecommerce/pull/1484/commits/0504a2bf7756fff2b18460a0245694069f4c3cc2
Screenshot 2024-02-13 at 3 49 03 PM

File: cms/templates/certificate_page.html (hunk 2 above)
Comment (asadali145) [reply cms/templates/certificate_page.html:47]: Updated in https://github.com/mitodl/bootcamp-ecommerce/pull/1484/commits/0504a2bf7756fff2b18460a0245694069f4c3cc2

Opera Snapshot_2024-02-13_154914_bc odl local
Opera Snapshot_2024-02-13_154954_bc odl local

**Review by arslanashraf7 — APPROVED** [review]
Looks good 👍  with one last question:

- Do we also need to update the favicon for this?

**Comment by asadali145** [comment]
> Do we also need to update the favicon for this?

I am not quite sure about that. Maybe @pdpinch or @mbilalmughal can comment on that?

Current favicon:
https://raw.githubusercontent.com/mitodl/bootcamp-ecommerce/master/static/images/favicon.ico

**Comment by mbilalmughal** [comment]
> > Do we also need to update the favicon for this?
> 
> I am not quite sure about that. Maybe @pdpinch or @mbilalmughal can comment on that?
> 
> Current favicon: https://raw.githubusercontent.com/mitodl/bootcamp-ecommerce/master/static/images/favicon.ico

@asadali145 We are using the black MIT favicon
https://ocw-hugo-themes-www-pr-1309--ocw-next.netlify.app/ - @ibrahimjaved12 recently used in OCW theme as well.

You can get the [Favicon asset](https://brand.mit.edu/downloads?downloads_keywords=&field_asset_type%5B0%5D=5&field_asset_type%5B1%5D=5) from here

We are going with the black background & white MIT logo, same as MIT [main website](https://www.mit.edu/) is using 

This one 
image

**Comment by asadali145** [comment]
@cachob What are your thoughts on the Favicon https://github.com/mitodl/bootcamp-ecommerce/pull/1484#issuecomment-1943364303?

**Comment by pdpinch** [comment]
> On the certificate, I think the MIT logo should be red. I think you can download the red logo from https://brand.mit.edu/system/files/downloads/MIT-logo.zip

We can't use that grey & red MIT logo on the certificate. I think the easiest thing to do is swap in a red logo from https://brand.mit.edu/system/files/downloads/MIT-logo.zip 

> What are your thoughts on the Favicon https://github.com/mitodl/bootcamp-ecommerce/pull/1484#issuecomment-1943364303?

Bilal is correct that we need to use the standard favicon. The black one is fine.

**Comment by mbilalmughal** [comment]
@asadali145 Here is the red MIT logo variation from the guidelines.

MIT-Logo-Red

**Comment by mbilalmughal** [comment]
That's how it will look.

Bootcamp-Certificate

**Comment by asadali145** [comment]
@arslanashraf7 Favicon and Certificate MIT logo are updated. Pls re-review.

**Review by arslanashraf7 — APPROVED** [review]
👍

**Comment by mbilalmughal** [comment]
The Logo Looks good to me. 👍 
The only concern is with the mobile version; the Print button seems broken. It's not related to the logo, so you should fix it in another story. (please see attach image)

image

**Pull request merged**